web-ui/src/App.css

```diff
@@ -1,12 +1,23 @@
-:root {
-  --oci-light-blue: #76c8d4;
-  --oci-dark-blue: #2c519e;
-  --oci-orange: #f8b576;
-  --nav-width: 150px;
+@import 'styles/_variables.css';
+
+.App {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  a,
+  a:visited {
+    color: var(--checkins-palette-primary-link);
```

Comment:
What do you think about replacing "checkins-palette" with just "checkins" everywhere just to shorten the names?

Reply:
Palette is something mui adds.